[Docs] Add Paperless Mobile app to docs (#2378)
paperless-ngx/paperless-ngx: docs/usage.md

App](https://github.com/bauerj/paperless_app) as well, which not only
has document upload, but also document browsing and download features.

> Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning,  management of labels and more.

Reviewer: ```suggestion
Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning, management of labels and more.
```

> Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), a feature rich Paperless mobile client supporting document upload, scanning,  management of labels and much more.

Reviewer: ```suggestion
Another option is [Paperless Mobile](https://github.com/astubenbord/paperless-mobile), an Android app that supports document upload, scanning,  management of labels and more.
```
